vobject/__init__.py

```diff
@@ -77,7 +77,13 @@
 """
 
 from .base import newFromBehavior, readOne, readComponents
-
+## TODO: if we import vcard here (or in the test code), the test code
+## breaks.  that should certainly be looked more into!  We most likely
+## should import vcard here, to make sure the __behaviorRegistry gets
+## populated correctly, and to make sure there won't be nasty
+## surprises that the library behaves differently dependent on weather
```

Comment:
Oups!  I wonder how many times I've done that mistake.

Some vcard-related tests in tests.py fails if importing vcard.  That's weird, isn't it?  Please try it for yourself.  Of course it could be just at my side, stale .pyc-files or whatnot - but then again, this is pretty consistent behaviour.  If I import vcard either from __init__.py or tests.py (or probably from just anywhere) some tests fail.  If I don't import, then they pass.